### PR TITLE
docs(samples): Add HTTP Task Sample with Authentication Token

### PR DESCRIPTION
**Pull request by averikitsch**
Adding Dockerfile to deploy to Cloud Run and sample with OIDC token for authentication.

- [ ] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by averikitsch** [comment]
Any ideas on why it's not finding the newly released OidcToken property?

File: samples/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+COPY package.json package*.json ./
+
+# Install production dependencies.
+RUN npm install --only=production
+
+# Copy local code to the container image.
+COPY . .
+
+# Run the web service on container startup.
+CMD [ "npm", "start" ]
```
Comment (JustinBeckwith) [inline samples/Dockerfile]: Well hello!  This is kinda new :) I'm assuming this is to better support cloud run?  Are we planning on dropping a Dockerfile in the root of all the sample repos?

Comment (averikitsch) [reply samples/Dockerfile]: Yes, it is. The docs associated with this will show how to used authenticated tasks with Cloud Run.

**Comment by averikitsch** [comment]
@JustinBeckwith I believe the new feature isn't being found. Though it looks like the oidc token, under target, property was added to the library, but it looks like npm hasn't had a new release yet.

**Comment by averikitsch** [comment]
Actually, I believe I need to add an actual service account.

**Pull request merged**